song/app.js

```diff
@@ -94,6 +98,54 @@ app.post("/content", async (req, res, next) => {
   );
   res.status(201).json({ message: "postCreated" });
 });
+// 좋아요 기능
+app.post("/like", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `
+    INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?)
+    `,
+    [userId, postId]
+  );
+
+  res.status(200).json({ message: "likeCreated" });
+});
+
+app.patch("/update", async (req, res, next) => {
+  const { content, userId } = req.body;
+  await appDataSource.query(
+    `UPDATE posts
+          SET 
+          content = ?
+          WHERE posts.user_id = ?
+          `,
+    [content, userId]
+  );
+  const update = await appDataSource.query(
+    `SELECT
+           users.id as userId,
+           users.name as userName,
+           posts.user_id as postingId,
+           posts.title as postingTitle,
+           posts.content as postingContent
+           FROM users LEFT JOIN posts ON users.id = posts.user_id     
+    `
+  );
+
+  res.status(200).json({ data: update });
```

Comment:
```suggestion
  return res.status(201).json({ data: update });
```
PATCH라는 method와 API 내부 로직을 생각해본다면 단순 응답을 나타내는 status code인 200보다 적절한 상태코드를 사용할 수 있을 것 같습니다.